### PR TITLE
Api keystone insecure bug

### PR DESCRIPTION
**Pull request by zhangjianweibj**
fix bug
https://github.com/monasca/monasca-docker/issues/497

insecure - Optional to specify if the request should be secure. Default is false.

### CODE REVIEW

File: monasca-api-python/Dockerfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ ENV CONFIG_TEMPLATE=true \
   KEYSTONE_ADMIN_PASSWORD=secretadmin \
   KEYSTONE_ADMIN_TENANT=admin \
   KEYSTONE_ADMIN_DOMAIN=default \
+  INSECURE=false \
```
Comment (matrixik) [inline monasca-api-python/Dockerfile]: Change variable name to `KEYSTONE_INSECURE`

File: monasca-api-python/README.md [hunk 2]
```diff
@@ -91,6 +91,7 @@ A number of environment variables can be passed to the container:
 | `KEYSTONE_ADMIN_USER`     | `admin`       | Keystone admin account user      |
 | `KEYSTONE_ADMIN_PASSWORD` | `secretadmin` | Keystone admin account password  |
 | `KEYSTONE_ADMIN_TENANT`   | `admin`       | Keystone admin account tenant    |
+| `INSECURE`                | `false`       | Optional to specify if the request should be secure |
```
Comment (matrixik) [inline monasca-api-python/README.md]: Change description to `Allow insecure Keystone connection`

File: monasca-api-python/Dockerfile (hunk 1 above)
Comment (zhangjianweibj) [reply monasca-api-python/Dockerfile]: ok,thanks.

**Comment by matrixik** [comment]
Sorry that this changes are still not merged.
There is problem with building monasca-api and now I'm talking with some people how to handle it properly.

**Pull request merged**